R/num_dist_by_cat.R

```diff
@@ -1,26 +1,94 @@
-#library( tidyverse)
-#library( cowplot)
-
 #' Create a histogram paired with a box plot.
 #'
-#' @param v_num Name of the column for the numeric variable.
-#' @param v_cat Name of the column for the categorical variable.
+#' @param num Name of the column for the numeric variable.
+#' @param group Name of the column for the categorical variable.
 #' @param data Target data frame for visualization.
 #' @param title_hist Title for the histogram.
 #' @param title_boxplot Title for the boxplot.
 #' @param lab_num Axis label for the numeric variable.
-#' @param lab_cat Axis label for the categorical variable.
+#' @param lab_group Axis label for the categorical variable.
 #' @param num_on_x Whether the numeric variable is put on the x-axis in the boxplot.
 #' @param stat Whether printing the test statistic and summary or not.
 #'
 #' @return A ggplot object consists of a histogram and a boxplot.
 #' @export
 #'
 #' @examples
```

Comment:
add one more space in front of @example. EXACTLY one space so it's indented compared to all the other lines